dataflow/src/main/java/org/checkerframework/dataflow/cfg/CFGBuilder.java

```diff
@@ -581,11 +586,23 @@ private static String uniqueName() {
      */
     protected static interface TryFrame {
         /**
-         * Given a type of thrown exception, add the set of possible control flow successor {@link
-         * Label}s to the argument set. Return true if the exception is known to be caught by one of
-         * those labels and false if it may propagate still further.
+         * Given a type of thrown exception, add to the argument set all possible pairs of the
+         * refined exception type {@link TypeMirror} and its control flow successor {@link Label} as
+         * a result of an exception thrown from a method invocation.
+         *
+         * <p>Return true if the exception is known to be caught by one of those labels and false if
+         * it may propagate still further.
+         *
+         * @param thrown the type of the exception that is declared to be thrown from a method
```

Comment:
Remove `.` at the end of the line. `@param` and `@return` should not write `.` at the end of the line. Please change all the usages in this PR.

Reply:
Do we also need to update this? Is it still accurate enough? https://github.com/opprop/checker-framework/pull/134/files#diff-852baab2528be0b3ea885d77fe6cdd8bL579